docs/ref/checks.txt

```diff
@@ -677,3 +678,14 @@ The following checks are performed on your URL configuration:
   able to reverse all URLs in this namespace.
 * **urls.E006**: The :setting:`MEDIA_URL`/ :setting:`STATIC_URL` setting must
   end with a slash.
+
+Staticfiles
+-----------
+
+The following checks verify that your :setting:`STATICFILES_DIRS` setting is
```

Comment:
I'd make the wording more general so that if more staticfiles checks are added later, we don't have to revise it.